lib/features/children/create_child/mixins/child_date_of_birth_validator.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:givt_app/l10n/l10n.dart';
+
+mixin ChildDateOfBirthValidator {
+  String? validateDateOfBirth({
+    required AppLocalizations locals,
```

Comment:
Not sure if this is going to be a good way of doing the localization as you have to pass it through a lot of objects

Reply:
https://nfcollect.atlassian.net/browse/KIDS-1514